Drivers/Inc/CAN.h

```diff
@@ -0,0 +1,54 @@
+#ifndef CAN_H__
+#define CAN_H__
+
+
+// Not sure if these are the ids we want but I took them from BPS as placeholders
+typedef enum {
+	TRIP = 0x02,
+    ALL_CLEAR = 0x101,
+    CONTACTOR_STATE = 0x102,
+    CURRENT_DATA = 0x103,
+    VOLT_DATA = 0x104,
+    TEMP_DATA = 0x105,
+    SOC_DATA = 0x106,
+    WDOG_TRIGGERED = 0x107,
+    CAN_ERROR = 0x108
+} CANId_t;

```

Comment:
Our relevant IDs should be on the CAN Bus IDs spreadsheet on the google drive. We send the Car State command and should be receiving the Motor Disable command. Also, we'll be receiving messages on the other CAN bus (CAN2) that we'll need to forward onto this bus using the same IDs. All of the IDs and command names/structures can be found on the Drive spreadsheet.